File: php/5kyu - A.php
//Link -> https://www.codewars.com/kata/526dbd6c8c0eb53254000110

function alphanumeric(string $s): bool {
    $c = str_split($s);

    foreach($c as $item){
        if(ctype_alnum($item) == false) return false;
    }

    return true;
}

File: scala/5kyu - A.scala
// Link -> https://www.codewars.com/kata/51edd51599a189fe7f000015

import scala.collection.mutable.ListBuffer

object Hello extends App{

  def solution(a: Array[Int], b: Array[Int]): Double = {
    var differences = new ListBuffer[Double]()

    for(i <- 0 until a.length){
      val instance: Double = scala.math.pow(b(i) - a(i), 2)
      differences += instance
    }

    differences.sum/differences.length
  }

}
